samples/secure-with-tls/docker-compose.yml

```diff
@@ -1,43 +1,37 @@
 version: "3.5"
 
-services:
-    grpc.client:
-        build: ./
-        environment:
-            # URL should match the DNS name in certificate and container name
-            - ESDB_CONNECTION_STRING=esdb://eventstoredb:2113?Tls=true
-        networks:
-            - esdb_network
-        depends_on:
-            eventstoredb:
-                condition: service_healthy
+networks:
+  default:
+    name: eventstore-network
 
-    eventstoredb:
-        image: eventstore/eventstore:20.10.0-buster-slim
-        # container_name should match the DNS name in certificate
-        container_name: eventstoredb
-        environment:
-            - EVENTSTORE_CLUSTER_SIZE=1
-            - EVENTSTORE_RUN_PROJECTIONS=All
-            - EVENTSTORE_ENABLE_EXTERNAL_TCP=true
-            - EVENTSTORE_ENABLE_ATOM_PUB_OVER_HTTP=true
```

Comment:
this line is needed for the stream browser in the UI to work. could be useful in the sample